src/scrabBag.js

```diff
@@ -0,0 +1,21 @@
+export default function scrabBag(theInput, tileDistro = {}) {
+  const inputArray = []
+
+  if (tileDistro === null) {
```

Comment:
*looks at code* whoa! yep it would never be null / i dont need this test. Thanks Jrob! 👍